Project.toml

```diff
@@ -4,9 +4,9 @@ authors = ["Johannes Schumann", "Tamas Gal"]
 version = "0.1.0"
 
 [deps]
-CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
```

Comment:
I already removed them in master, so don't be surprised. They broke the Travis build due to whatever weird error ;)

Reply:
I've seen your commit, but as it affects just two lines I thought a rebase is kinda over the top ;-)

Reply:
It seems that it has no merge conflicts, so all good ;)